packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -105,7 +103,7 @@ export const messages = {
   force_lightning_event_create_text: 'SFDX: Create Lightning Event',
   force_lightning_interface_create_text: 'SFDX: Create Lightning Interface',
   force_source_status_local_text: 'SFDX: View Local Changes',
-  force_source_status_remote_text: 'SFDX: View Changes in Default Scratch Org',
+  force_source_status_remote_text: 'SFDX: View Changes in Default Org',
```

Comment:
The commands whose messages are on lines 66 through 106 (`force_source_pull_default_scratch_org_text` through `force_source_status_remote_text`)work only with scratch orgs, so I'd leave them unchanged.